ag/ag.proto

```diff
@@ -199,6 +199,25 @@ message Issue {
     uint64 issueNumber  = 4;  // Issue number on scm. Needed for associating db issue with scm issue
 }
 
+message PullRequest {
+    enum Stage {
+        NONE    = 0;
+        DRAFT   = 1;
+        REVIEW  = 2;
+        APPROVED = 3;
+    }
+    uint64 ID                   = 1;
+    // TODO(meling): When a naming convention has been decided, change this field's name.
+    uint64 externalRepositoryID = 2; // Represents the external repository ID
+    uint64 taskID               = 3; // foreign key
+    uint64 issueID              = 4; // foreign key
+    uint64 userID               = 5; // The user who owns this PR
+    uint64 commentID            = 6; // GitHub ID of the comment used for automatic feedback
```

Comment:
I guess this one should also be renamed to `scmCommentID` and the comment should probably not mention GitHub ; presumably we could implement a similar thing with GitLab in the future.